go/libraries/doltcore/sqle/tables.go

```diff
@@ -1778,7 +1796,8 @@ func (t *AlterableDoltTable) CreateIndex(ctx *sql.Context, idx sql.IndexDef) err
 	for _, idxCol := range idx.Columns {
 		col := t.DoltTable.sqlSch.Schema[t.DoltTable.sqlSch.IndexOfColName(idxCol.Name)]
 		if idxCol.Length > 0 && sql.IsText(col.Type) {
-			return sql.ErrUnsupportedIndexPrefix.New(col.Name)
+			// TODO: delete me
+			//return sql.ErrUnsupportedIndexPrefix.New(col.Name)
```

Comment:
Is this supposed to have been deleted? If so the `if` statement can also be removed.